core/lib/multivm/src/versions/vm_1_3_2/transaction_data.rs

```diff
@@ -170,7 +170,7 @@ impl TransactionData {
         encode(&[Token::Tuple(vec![
             Token::Uint(U256::from_big_endian(&self.tx_type.to_be_bytes())),
             Token::Address(self.from),
-            Token::Address(self.to),
+            Token::Address(self.to.unwrap_or_default()),
```

Comment:
You mean to apply this change for older VMs? At least for `vm_latest`, we can have a `None` at this point, so we need to take the default `U256` to avoid panicking.

Reply:
> This method is to be used only in tests, when we want to bypass the checks imposed

I don't think the `None` scenario is implied in tests. On the other hand, I don't think it's a problem.

Reply:
Correct me if I'm wrong, but this function `abi_encode_with_custom_factory_deps` is used every time we send a new transaction since it's called by the `BootloaderTx::new()` function, which happens every time we push a new transaction into the bootloader memory.

Changing this line to an `unwrap()` in `vm_latest` causes a panic every time I try to deploy an EVM contract. 
Maybe I'm missing something here, so please let me know.